src/mscorlib/corefx/System/Globalization/CultureData.Unix.cs

```diff
@@ -180,8 +198,10 @@ private static CultureInfo GetUserDefaultCulture()
 
         private static bool IsCustomCultureId(int cultureId)
         {
-            // TODO: Implement this fully.
-            return false;
+            const int LOCALE_CUSTOM_DEFAULT = 0x0c00;
+            const int LOCALE_CUSTOM_UNSPECIFIED = 0x1000;
+
+            return (cultureId == LOCALE_CUSTOM_DEFAULT || cultureId == LOCALE_CUSTOM_UNSPECIFIED);
```

Comment:
I am going to defer commenting on this until I add support for adding additional locale validation which will test "custom locales" which have a valid signature but don't exist.